R CODE FINAL Alexander Shemetev

```diff
@@ -0,0 +1,791 @@
+  tn2 = x2[2,2]
+  tn3 = x3[2,2]
+  tn4 = x4[2,2]
+  tn5 = x5[2,2]
+  tn6 = x6[2,2]
+  tn7 = x7[2,2]
+  tn8 = x8[2,2]
+  tn9 = x9[2,2]
+  tn10 = x10[2,2]
+  tn11 = x11[2,2]
+  tn12 = x12[2,2]
+  tn13 = x13[2,2]
+  tn14 = x14[2,2]
+  tn15 = x15[2,2]
+  tn16 = x16[2,2]
+  tn17 = x17[2,2]
+  tn18 = x18[2,2]
+  tn19 = x19[2,2]
+  
+  fp1 = x1[1,2]
+  fp2 = x2[1,2]
+  fp3 = x3[1,2]
+  fp4 = x4[1,2]
+  fp5 = x5[1,2]
+  fp6 = x6[1,2]
+  fp7 = x7[1,2]
+  fp8 = x8[1,2]
+  fp9 = x9[1,2]
+  fp10 = x10[1,2]
+  fp11 = x11[1,2]
+  fp12 = x12[1,2]
+  fp13 = x13[1,2]
+  fp14 = x14[1,2]
+  fp15 = x15[1,2]
+  fp16 = x16[1,2]
+  fp17 = x17[1,2]
+  fp18 = x18[1,2]
+  fp19 = x19[1,2]
+  
+  fn1 = x1[2,1]
+  fn2 = x2[2,1]
+  fn3 = x3[2,1]
+  fn4 = x4[2,1]
+  fn5 = x5[2,1]
+  fn6 = x6[2,1]
+  fn7 = x7[2,1]
+  fn8 = x8[2,1]
+  fn9 = x9[2,1]
+  fn10 = x10[2,1]
+  fn11 = x11[2,1]
+  fn12 = x12[2,1]
+  fn13 = x13[2,1]
+  fn14 = x14[2,1]
+  fn15 = x15[2,1]
+  fn16 = x16[2,1]
+  fn17 = x17[2,1]
+  fn18 = x18[2,1]
+  fn19 = x19[2,1]
+  
+  
+  
+  accuracy0.05 = (tp1+fn1)/(tp1+fn1+fp1+tn1)
+  accuracy0.1 = (tp2+fn2)/(tp2+fn2+fp2+tn2)
+  accuracy0.15 = (tp3+fn3)/(tp3+fn3+fp3+tn3)
+  accuracy0.2 = (tp4+fn4)/(tp4+fn4+fp4+tn4)
+  accuracy0.25 = (tp5+fn5)/(tp5+fn5+fp5+tn5)
+  accuracy0.3 = (tp6+fn6)/(tp6+fn6+fp6+tn6)
+  accuracy0.35 = (tp7+fn7)/(tp7+fn7+fp7+tn7)
+  accuracy0.4 = (tp8+fn8)/(tp8+fn8+fp8+tn8)
+  accuracy0.45 = (tp9+fn9)/(tp9+fn9+fp9+tn9)
+  accuracy0.5 = (tp10+fn10)/(tp10+fn10+fp10+tn10)
+  accuracy0.55 = (tp11+fn11)/(tp11+fn11+fp11+tn11)
+  accuracy0.6 = (tp12+fn12)/(tp12+fn12+fp12+tn12)
+  accuracy0.65 = (tp13+fn13)/(tp13+fn13+fp13+tn13)
+  accuracy0.7 = (tp14+fn14)/(tp14+fn14+fp14+tn14)
+  accuracy0.75 = (tp15+fn15)/(tp15+fn15+fp15+tn15)
+  accuracy0.8 = (tp16+fn16)/(tp16+fn16+fp16+tn16)
+  accuracy0.85 = (tp17+fn17)/(tp17+fn17+fp17+tn17)
+  accuracy0.9 = (tp18+fn18)/(tp18+fn18+fp18+tn18)
+  accuracy0.95 = (tp19+fn19)/(tp19+fn19+fp19+tn19)
+  
+  
+  acura=c(accuracy0.05, accuracy0.1, accuracy0.15, accuracy0.2, accuracy0.25, accuracy0.3, accuracy0.35, accuracy0.4, accuracy0.45, accuracy0.5, accuracy0.55, accuracy0.6, accuracy0.65, accuracy0.7, accuracy0.75, accuracy0.8, accuracy0.85, accuracy0.9, accuracy0.95 )
+  
+  
+  macura = max(acura)
+  #description 
+  # words[which (nchar(words) == max(nchar(words)))]
+  # which(v == max(v))
+  #names(v)[v == "NY"] # extract the names, subset by equality to NY
+  # or
+  #names(which(v == "NY")) # extract entries that == NY and get names
+  #acurainfo = c(which(macura == max(acura)))
+  acurainfo = c(0.05, 0.1, 0.15, 0.2, 0.25, 0.3, 0.35, 0.4, 0.45, 0.5, 0.55, 0.6, 0.65, 0.7, 0.75, 0.8, 0.85, 0.9, 0.95)
+  #list(macura, acurainfo, acura)
+  data.frame(macura, acurainfo, acura)
+  # TryCatch - for dummies - they have no 1 and 99 percentile - causes error. 
+  #tryCatch(list(description,quantilesxx, sdex, summix, nobs), error=function(e) list(description, sdex, summix, nobs))
+  #x
+}
+var_logit(bankloans_new$income)
```

Comment:
super ugly solution. It does work, yet it can be done in a single for loop without all this lengthy copy-paste type expressions
Moreover, it is not a function of x